src/main/python/segment_collection/utils.py

```diff
@@ -0,0 +1,54 @@
+import jnius_config
```

Comment:
And you're planning on moving common code to something like `src/main/python/io/anserini/collection/`? We probably want to mirror the java package structure?